bundles/org.openhab.core.karaf/src/main/java/org/openhab/core/karaf/internal/KarafAddonService.java

```diff
@@ -13,7 +13,13 @@
 package org.openhab.core.karaf.internal;
 
 import static java.util.Map.entry;
-import static org.openhab.core.addon.AddonType.*;
```

Comment:
Probably done automatically by the IDE, but no need to change this for a static import.